x-pack/plugins/cloud_security_posture/public/pages/findings/latest_findings_by_resource/resource_findings/use_resource_findings.ts

```diff
@@ -46,7 +50,8 @@ const getResourceFindingsQuery = ({
   size,
   pitId,
   sort,
-}: UseResourceFindingsOptions & { pitId: string }): estypes.SearchRequest => ({
+  aggs,
```

Comment:
instead of passing `aggs`, we can pass the values needed for the query, same as `resourceId`, and then generate all aggregations query (including `count`).